dotcom-rendering/playwright/tests/commercial.e2e.spec.ts

```diff
@@ -15,10 +13,10 @@ test.describe('Commercial E2E tests', () => {
 		await cmpAcceptAll(page);
 
 		// We are excluding survey slot as they can be switched off
-		await expectToExist(page, '.js-ad-slot:not([data-name="survey"])', 15);
+		await expectToExist(page, '.js-ad-slot:not([data-name="survey"])', 16);
 
 		// Check all inline slots are present
-		for (const i of Array(11).keys()) {
+		for (const i of Array(12).keys()) {
```

Comment:
Minor but while we're here. Could this value be calculated from the total number of slots we expect? For example:

```js
const totalSlots = 16;
const fixedSlots = 4;
const inlineSlots = totalSlots - fixedSlots;
```

Reply:
nice thanks Ravi